scripts/hookup.py

```diff
@@ -27,7 +27,7 @@
     parser.add_argument('--show-levels', help="Show power levels if enabled (and able) [False]", dest='show_levels', action='store_true')
     parser.add_argument('--show-ports', help="Show ports on hookup.", dest='show_ports', action='store_true')
     parser.add_argument('--show-revs', help="Show revs on hookup.", dest='show_revs', action='store_true')
-    parser.add_argument('--show-cache-file-info', help="Shows information about the hookup cache file.", dest='cache_info', action='store_true')
+    parser.add_argument('--show-cache-info', help="Shows information about the hookup cache file.", dest='cache_info', action='store_true')
```

Comment:
I meant the `dest` setting here, not the command-line argument name (although I think the shorter argument name is probably an improvement too).